WebCryptoAPI/wrapKey_unwrapKey/wrapKey_unwrapKey.https.any.js

```diff
@@ -127,112 +129,117 @@
             formats = ["raw", "jwk"]
         }
 
-        return Promise.all(formats.map(function(fmt) {
-            var originalExport;
-            return subtle.exportKey(fmt, toWrap.key).then(function(exportedKey) {
-                originalExport = exportedKey;
-                const isPossible = wrappingIsPossible(originalExport, wrapper.parameters.name);
-                promise_test(function(test) {
+        formats.map(function(fmt) {
+            var isPossible;
+            promise_test(async() => {
+                try {
+                    var originalExport;
+                    var exportedKey = await subtle.exportKey(fmt, toWrap.key);
+                    originalExport = exportedKey;
+                    isPossible = wrappingIsPossible(originalExport, wrapper.parameters.name);
```

Comment:
Is it possible (pun not intended) to pull this variable out of this test, into the setup? It seems a bit ugly (and possibly unreliable?) to rely on a value computed in one test in another test...

And then, instead of skipping inside the test, perhaps that particular test case can just not be created?

Reply:
I'll try; I just wanted to made the minimum changes to this already complex test, but I'll try. I agree this "isPossible" approach of creating "empty" Promises is ugly.